### PR TITLE
Finish Pokemon List

### PR DESCRIPTION
**Pull request by dennism1997**
The template wasn't necessary after all. But I do think this code is more clean and easier to maintain.
This PR also implements less and all the required dependencies.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -24,7 +25,7 @@ const srcs = {
     buildArtefacts: 'build/**/*',
     scripts: 'src/scripts/**/*.ts',
     html: ['src/*.html', 'src/templates/*.html'],
-    styles: 'src/styles/**/*.css',
+    styles: 'src/styles/**/*.less',
```
Comment (Ishadijcks) [inline gulpfile.js]: All css is also less right?

**Comment by Ishadijcks** [comment]
image
On half screen the list overflows.

Also it seems like the attack and level listeners aren't working?

**Comment by dennism1997** [comment]
On half screen the columns should collapse right?

File: gulpfile.js (hunk 1 above)
Comment (dennism1997) [inline gulpfile.js]: yes. But also all less is css

**Comment by dennism1997** [comment]
The listeners did work already I think. Just did some refactoring. The values don't get computed yet so that's why they don't change

**Comment by Ishadijcks** [comment]
Ah, I see.
Note that attackBonus is the bonus gained from breeding/items.
And it's attack is calculated using the base Pokémon and the above factors.

**Pull request merged**